README.md

```diff
@@ -23,6 +42,11 @@ gem 'smart_proxy_remote_execution_ssh', :path => '../smart_proxy_remote_executio
 gem 'smart_proxy_remote_execution_ssh_core', :path => '../smart_proxy_remote_execution_ssh'
 END
 ```
+Rerun forman and seed database
+```
+bundle exec rake db:seed
```

Comment:
I wouldn't even include this here, it's not necessary for the smart proxy component.